client/src/includes/dialog.js

```diff
@@ -6,7 +6,7 @@ export const dialog = (
   dialogs.forEach((template) => {
     const html = document.documentElement;
     const templateContent = template.content.firstElementChild;
-    document.body.appendChild(templateContent);
+    template.insertAdjacentElement('afterend', templateContent);
```

Comment:
Nice! I think this would warrant a code comment as the implications of this are big, even if this new setup makes much more sense than the previous.

Reply:
I believe this is causing issues with the Set privacy dialog, which is now rendering inside the side panel rather than as a whole-page takeover.